src/umpire/ResourceManager.hpp

```diff
@@ -108,7 +112,7 @@ class ResourceManager {
   /*!
    * \brief Construct a new Allocator.
    */
-  template <typename Strategy, bool introspection = true, typename... Args>
+  template <typename Strategy, bool introspection = false, typename... Args>
```

Comment:
I didn't know we were changing the default for enabling introspection.  Was this intentional?